### PR TITLE
fix direction from

### PR DESCRIPTION
**Pull request by Skalii**

### CODE REVIEW

**Review by tigrim — APPROVED** [review]

**Comment by alex-osm** [comment]
We need confirmation that the changes in the request correspond to the logic requested in the issue.

**Comment by Skalii** [comment]
<table style="width:100%">
  <tr>
    <th>Android</th>
    <th>iOS</th>
  </tr>
  <tr>
    <td> </td>
    <td> </td>
  </tr>
</table>

File: Sources/Controllers/RouteInfoMenu/OARouteInfoView.h [hunk 1]
```diff
@@ -18,6 +18,7 @@
 @property (weak, nonatomic) IBOutlet UIButton *goButton;
 @property (weak, nonatomic) IBOutlet UIView *contentContainer;
 @property (weak, nonatomic) IBOutlet UIView *statusBarBackgroundView;
+@property (nonatomic) BOOL selectFromMapTouch;
```
Comment (alex-osm) [inline Sources/Controllers/RouteInfoMenu/OARouteInfoView.h:21]: Not clear why this field added. It's name is confusing. Looks like it should be dropped.

File: Sources/Controllers/Panels/OAMapPanelViewController.mm [hunk 2]
```diff
@@ -821,6 +821,12 @@ - (void) closeRouteInfo
     [self closeRouteInfo:YES onComplete:nil];
 }
 
+- (void)closeRouteInfoByTappingOnMap
```
Comment (alex-osm) [inline Sources/Controllers/Panels/OAMapPanelViewController.mm:824]: Why you handle close by tapping. What if menu closed by cancel button?

**Pull request merged**